File: KoTi/ClientApp/src/model/Stats.ts
interface Stats {
    totalPictures: number;
    totalPicturesWithNoLocation: number;
    totalPlaces: number;
    totalPosts: number;
    totalArticles: number;
    databaseLastPublishedAt: string;
    databaseSize: number;
    s3Bucket: string;
}

export default Stats;

File: KoTi/ResponseModels/Stats.cs
namespace KoTi.ResponseModels;

public class Stats
{
    public int TotalPictures { get; set; }

    public int TotalPicturesWithNoLocation { get; set; }

    public int TotalPlaces { get; set; }
    
    public int TotalPosts { get; set; }
    
    public int TotalArticles { get; set; }
    
    public DateTime DatabaseLastPublishedAt { get; set; }
    
    public long DatabaseSize { get; set; }

    public string S3Bucket { get; set; } = "";
}